Text runner is failing to start with illegal option error

On my macOS machine I am unable to start text-runner because:

```
% npx text-runner
Need to install the following packages:
  text-runner
Ok to proceed? (y) y
readlink: illegal option -- f
usage: readlink [-n] [file ...]
node:internal/modules/cjs/loader:936
  throw err;
  ^

Error: Cannot find module '/Users/foobar/workspace/dist/index.js'
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12)
    at node:internal/main/run_main_module:17:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}

Node.js v17.4.0
```

This appears to be caused by incorrect usage of `readlink` on macOS:

```
% readlink -f
readlink: illegal option -- f
usage: readlink [-n] [file ...]
```

--- a/text-runner-cli/bin/text-run
+++ b/text-runner-cli/bin/text-run
@@ -1,10 +1,3 @@
-#!/usr/bin/env bash
+#!/usr/bin/env node
 
-# NOTE: Yarn creates symlinks of binaries in workspaces.
-#       We need to resolve them to find the correct path to index.js
-if [[ -L $0 ]]; then
-  cliPath=$( dirname "$(realpath $0)" )
-else
-  cliPath="$( dirname "${BASH_SOURCE[0]}")"
-fi
-node --enable-source-maps $cliPath/../dist/index.js "$@"
+require("../dist/index")

--- a/text-runner-cli/bin/text-run
+++ b/text-runner-cli/bin/text-run
@@ -3,7 +3,7 @@
 # NOTE: Yarn creates symlinks of binaries in workspaces.
 #       We need to resolve them to find the correct path to index.js
 if [[ -L $0 ]]; then
-  cliPath=$( dirname "$(readlink -f $0)" )
+  cliPath=$( dirname "$(realpath $0)" )
 else
   cliPath="$( dirname "${BASH_SOURCE[0]}")"
 fi

